cogs/send_get_roles_reminders.py

```diff
@@ -93,6 +93,7 @@ async def send_get_roles_reminders(self) -> None:
                 "Serious Talk",
                 "Housing",
                 "Gaming",
+                "Pets",
```

Comment:
has this channel actually been made yet and if not is this gonna cause an error if it's merged before the channel exists?

Reply:
ah yh my bad